packages/commonwealth/client/scripts/models/Thread.ts

```diff
@@ -137,7 +137,7 @@ export class Thread implements IUniqueId {
   public readonly slug = ProposalType.Thread;
   public readonly url: string;
   public readonly versionHistory: VersionHistory[];
-  public readonly chain: string;
+  public readonly community_id: string;
```

Comment:
shouldn't we use `communityId` instead of `community_id`? Not sure if we have defined standard though as I can see actually both snake_case and camelCase in the file

Reply:
Fixed